setup/local/README.md

```diff
@@ -264,34 +264,53 @@ git clone https://github.com/sse-secure-systems/connaisseur.git
 cd connaisseur
 ```
 
-In there, you will find a `helm` directory, which holds a `values.yaml` file, which needs to be configured. Change the `notary` section to reflect the appropriate values. In `notary.selfsignedCert` you have to put the certificate from `harbor-ca.crt`.
+In there, you will find a `helm` directory, which holds a `values.yaml` file, which needs to be configured. Add a new notary configuration with all appropriate values. The `selfsignedCert` filed should be filled with the contents of the `harbor-ca.crt` from earlier.
```

Comment:
```suggestion
In there, you will find a `helm` directory, which holds a `values.yaml` file, which needs to be configured. Add a new notary configuration with all appropriate values. The `selfsignedCert` field should be filled with the contents of the `harbor-ca.crt` from earlier.
```

Reply:
resolved in 3b7d64cd6e6c5c09c2f7370d57ddb8a3b6c8869c